<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visitsSet</class>
 <widget class="QDialog" name="visitsSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Lekarz</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_doctor"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pacjent</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Godzina</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_procedure"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox_patient"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Zabieg</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QTimeEdit" name="timeEdit_visit"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_accept">
     <property name="text">
      <string>Zatwierd≈∫</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
